### PR TITLE
Update the demo video in README.md

### PR DESCRIPTION
**Pull request by ly015**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,22 +20,15 @@ It is a part of the [OpenMMLab project](https://github.com/open-mmlab).
...
 
 <div align="center">
-    <img src="demo/resources/demo_coco.gif" width="600px" alt><br>
-    COCO 17-keypoint pose estimation
-</div>
-<div align="center">
-    <img src="https://user-images.githubusercontent.com/9464825/95552839-00a61080-0a40-11eb-818c-b8dad7307217.gif" width="600px" alt><br>
-
-133-keypoint whole-body pose estimation ([full HD version](https://www.youtube.com/watch?v=pIJpQg8mXUU))
+    <img src="https://openmmlab.oss-cn-hangzhou.aliyuncs.com/mmpose/demo/mmpose_full_demo_v2.1s.gif" width="600px" alt><br>
```
Comment (innerlee) [inline README.md:23]: The gif is 85M, and is too big for most users.
Suppose on is using cellphone, then his/her data will got some loss just viewing this page.

I suggest post this as a video file, as showed ealier

File: README.md [hunk 2]
```diff
@@ -20,22 +20,15 @@ It is a part of the [OpenMMLab project](https://github.com/open-mmlab).
...
 </div>
+
 <div align="center">
-    <img src="https://user-images.githubusercontent.com/11788150/114201893-4446ec00-9989-11eb-808b-5718c47c7b23.gif" width="600px" alt><br>
-    2D animal_pose estimation
-</div>
-<div align="center">
-    <img src="https://user-images.githubusercontent.com/15977946/118820606-02df2000-b8e9-11eb-9984-b9228101e780.gif" width="600px" alt><br>
-    3D human pose estimation
+    <img src="https://user-images.githubusercontent.com/28900607/121288285-b8fcbf00-c915-11eb-98e4-ba846de12987.gif" width="600px" alt><br>
```
Comment (innerlee) [inline README.md:29]: I do not understand why we need two gifs

File: README.md [hunk 3]
```diff
@@ -20,22 +20,15 @@ It is a part of the [OpenMMLab project](https://github.com/open-mmlab).
...
-    <img src="demo/resources/demo_coco.gif" width="600px" alt><br>
-    COCO 17-keypoint pose estimation
-</div>
-<div align="center">
-    <img src="https://user-images.githubusercontent.com/9464825/95552839-00a61080-0a40-11eb-818c-b8dad7307217.gif" width="600px" alt><br>
-
-133-keypoint whole-body pose estimation ([full HD version](https://www.youtube.com/watch?v=pIJpQg8mXUU))
+    <img src="https://openmmlab.oss-cn-hangzhou.aliyuncs.com/mmpose/demo/mmpose_full_demo_v2.1s.gif" width="600px" alt><br>
 
+[Watch full demo video on GitHub](https://youtu.be/nFcZ2H1Ix3w)
```
Comment (innerlee) [inline README.md:25]: on ??Github??

File: README.md (hunk 1 above)
Comment (ly015) [reply README.md:23]: Still trying to find a proper way to embed the demo. Video files cannot be embedded into GitHub markdown.

Comment (innerlee) [reply README.md:23]: https://github.com/open-mmlab/mmpose/issues/429#issuecomment-858308186

**Review by innerlee — APPROVED** [review]

**Pull request merged**